tests/MoBi.Tests/Presentation/ParameterDTOSpecs.cs

```diff
@@ -18,13 +18,14 @@ protected override void Context()
       }
    }
 
-   public class When_creating_a_parameter_dto_for_a_constant_aprameter_with_an_undefined_value : concern_for_ParameterDTO
+   public class When_creating_a_parameter_dto_for_a_constant_a_parameter_with_an_undefined_value : concern_for_ParameterDTO
```

Comment:
When_creating_a_parameter_dto_for_a_constant_a_parameter_with_an_undefined_value
When_creating_a_parameter_dto_for_a_constant_parameter_with_an_undefined_value

is it supposed to be constant_parameter instead of constant_a_parameter?